test/integration/scheduler_perf/util.go

```diff
@@ -70,10 +85,18 @@ func mustSetupScheduler() (util.ShutdownFunc, coreinformers.PodInformer, clients
 		Burst:         5000,
```

Comment:
One question: should we honor QPS and other settings in given `config`? If so, `config` is unversioned, which implied each field employs a concrete value, then if `config` is not nil, how can we decide to derive the values in `config` vs. apply the values specified here (5000)?

Reply:
we can take this as a default value, and override this value if CC has a value defined, maybe we can drop the TODO item here and address it in the v1.22?

Reply:
mark this resolved, feel free to drop any comments anyway.